lib/utils/timetable.ts

```diff
@@ -59,14 +63,23 @@ export const resolveTimetableEventsCSV = (
     );
     if (!formEvent) {
       // This shouldn't ever happen
-      throw Error("Form event not found");
+      throw Error(`event ${loadedEvent.developmentPlanEvent} not found`);
+    }
+    const currentDate = getFormattedDate();
+
+    const isStatusChangeEvent = statusChangeEvents.some(
+      (event) => event === formEvent.developmentPlanEvent
+    );
```

Comment:
The types get a little funny because `statusChangeEvents` is a `const` (as opposed to an array of `TimetableEventKey` but, with some casting, we should be able to use `.includes` here instead of `.some`. Since this seems to be used in a few places, maybe we should make a type guard (e.g. after the type definition in `constants.ts`):

```ts
export const isStatusChangeEventKey = (
  timetableEventKey: TimetableEventKey
): timetableEventKey is StatusChangeEventKey =>
  (statusChangeEvents as unknown as TimetableEventKey[]).includes(
    timetableEventKey
  );
```

You can also cast `timetableEventKey` to `StatusChangeEventKey` (instead of the `statusChangeEvents` cast) but that's not technically true, whereas a status change event _is_ a timetable event (even if the types don't like it, meaning we need a cast to `unknown` first).

Reply:
Thanks Ronan :) Done!